docs/examples/validators_allow_reuse.py

```diff
@@ -0,0 +1,25 @@
+from pydantic import BaseModel, validator
+
+def normalize(name: str) -> str:
+    return " ".join((word.capitalize()) for word in name.split(" "))
```

Comment:
```suggestion
    return ' '.join(word.capitalize() for word in name.split(' '))
```

single quotes throughout please.